internal/service/advancedclustertpf/plural_data_source.go

```diff
@@ -77,7 +77,7 @@ func (d *pluralDS) readClusters(ctx context.Context, diags *diag.Diagnostics, pl
 		if diags.HasError() {
 			return nil
 		}
-		if extraInfo.AsymmetricShardUnsupported && !useReplicationSpecPerShard {
+		if extraInfo.ForceLegacySchemaFailed {
```

Comment:
Maybe do a `tflog.Info`? Alternatively, a diags.AddWarning, but that might be a bit overkill?

Reply:
I wouldn't do AddWarning as we're not doing it either SDKv2. Also I wouldn't even send info as this is a normal use case and logs can get very verbose